main/templates/main/index.html

```diff
@@ -65,34 +63,51 @@
   <a class="home-button" href="{% url 'our_team' %}">Our awesome team!</a>
 </section>
 
+<!--Projects-->
 <section id="section-projects">
-  <div class="container">
-    <h2>Our Projects</h2>
-    <div class="row">
-    {% for project in projects %}
-      <div class="col-sm-4">
-        <a href="{% url 'project_details' project.id %}">
-          <div class="project-card">
-            {% if project.image %}
-              <img src="{{project.image.url}}" alt="{{project.title}}" class="project-pic">
-            {% endif %}
-            <div class="project-text">
-              <div class="project-title">
-                {{project.title}}
-              </div>
-              <div class="project-short-desc">
-                {{project.shortDesc}}
+  <a name="projects">
```

Comment:
I will change it to the HTML 5 version in the next update (with complete jump + hover links)